### ISSUE TITLE
Instructions on how to contribute? (+ Addon corrupt - Firefox only)

### ISSUE THREAD

**Issue by flleeppyy** [insin/tweak-new-twitter#35]
This is my first time working with a browser extension, and I want to contribute to it.
As this is my first time, I don't know entirely how to get this to work with Firefox when building, so I try to go about it normally like any other Javascript project.
I run `npm install`, installs everything needed for `web-ext`, then I run `npm build`. I drop it into my extensions page in Firefox developer edition, but when I drop in the artifact it says the addon is corrupt. This doesn't make much sense to me considering I haven't made any changes to the script at all

![image](https://user-images.githubusercontent.com/18307183/114118808-50a24a80-989e-11eb-83b3-2bf29e83adf9.png)

**Comment by insin** [comment]
I'll add a `CONTRIBUTING.md` soon.

TL;DR: to run it in Firefox use `npm start`, to run it in Chrome, use `npm run chrome` - web-ext will then create a temporary profile and run the extension in it, reloading it when you save changes.
